src/pages/mining.md

```diff
@@ -23,6 +23,33 @@ If you want to learn more about the technical details of minig, please review th
 [@page-reference | inline]
 | /stacks-blockchain/mining
 
+## Running bitcoind locally
+
+To participate as a miner on Xenon, you must have access to a testnet bitcoin node. One way to accomplish this is to run bitcoind locally. You'll need a computer with ~10-GB disk space.
+
+First, download the bitcoind software for your platform from https://bitcoin.org/en/download.
+
+Next, start bitcoind with the following configuration:
+
+```
+server=1
+rpcuser=your-bitcoind-username
+rpcpassword=your-bitcoind-password
+testnet=1
+txindex=0
+listen=1
+rpcserialversion=0
+maxorphantx=1
+banscore=1
+
+[test]
+bind=0.0.0.0:18443
+rpcbind=0.0.0.0:18444
+rpcport=18443
```

Comment:
We're now using the standard BTC testnet ports for our deployment. Since we're allowing users running a follower to connect to our BTC node's p2p port, we should ensure users running a miner use the same BTC ports for uniformity.
```suggestion
bind=0.0.0.0:18333
rpcbind=0.0.0.0:18332
rpcport=18332
```

Reply:
Sure, happy to change it. I was trying to minimize diff from the Krypton config.